frameworks/GAMA/setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+shopt -s expand_aliases
```

Comment:
not sure this is still useful, not using aliases anymore (they're not reliable)

Reply:
Will remove, I had this copied over from a the RandomForest `setup.sh`